### DISCUSSION TITLE
[CFP] 43 spotkanie Microsoft Azure User Group Poland

### DISCUSSION THREAD

**Discussion by kaluzaaa** [AzureCommunityPL/group#65] [Pomoc]
No to lecimy z CFP na kolejny meetup (14.10). Zgłaszanie sesji otwarte do 20.09 do 19:00

Zapraszamy każdego chętnego! Meetup to świetne miejsce na start!

Jeśli to Twoja pierwsza prezentacja to chętnie pomożemy :-)

[Link do zgłoszeń](https://azurecommunitypl.typeform.com/to/ZTDQWviM)

**Comment by mgrabarz** [comment]
Podbijam temat. Zostało jeszcze kilka dni na zgłoszenia.
